lib/lookup_users.rb

```diff
@@ -21,21 +21,26 @@ def self.by_email_or_username(email_or_username)
       return [matches.first] if matches.one?
     end
 
+    return self.by_verified_email(email_or_username)
+  end
+
+  def self.by_verified_email(email)
     # Case-sensitive email search
     ContactInfo.verified
-               .where(value: email_or_username)
+               .where(value: email)
```

Comment:
wow I'm guessing this code was not being used :-)

Reply:
Oh that was because I added a new `by_verified_email(email)` to  lookup_users